<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>641</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabDailyRevenue">
     <attribute name="title">
      <string>Ngày</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>từ ngày</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>đến ngày</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditFrom">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditTo">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDailyRevenue">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Thống kê</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableDailyRevenue">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>521</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotalDailyRevenue">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMonthlyRevenue">
     <attribute name="title">
      <string>Tháng</string>
     </attribute>
     <widget class="QLabel" name="lblSelectMonth">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chọn tháng</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSelectYearMonth">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chọn năm</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxMonth">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>tháng 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 12</string>
       </property>
      </item>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEditYearMonth">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMonthlyRevenue">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Thống kê</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableMonthlyRevenue">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>431</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotalMonthlyRevenue">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>490</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabYearlyRevenue">
     <attribute name="title">
      <string>năm</string>
     </attribute>
     <widget class="QLabel" name="lblSelectYear">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chọn năm</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="spinBoxYear">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnYearlyRevenue">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Thống kê</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableYearlyRevenue">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotalYearlyRevenue">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>510</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMonthlyTicketRevenue">
     <attribute name="title">
      <string>vé tháng, năm</string>
     </attribute>
     <widget class="QLabel" name="lblSelectMonth_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chọn tháng</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSelectYearMonth_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chọn năm</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="spinBoxYearMonthlyTicket">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxMonthMonthlyTicket">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>tháng 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tháng 12</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblTotalMonthlyTicketRevenue">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>520</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMonthlyTicketRevenue">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Thống kê</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableMonthlyTicketRevenue">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thống kê doanh thu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
